checker/tests/nullness-extra/issue5174/Issue5174.out

```diff
@@ -413,12 +413,12 @@ Before: InitializationStore#93(
 ~~~~~~~~~
 o   [ VariableDeclaration ]
 (this)   [ ImplicitThis ]
-(this).f   [ FieldAccess ]    > CFAV{@Initialized, Object}
-o = (this).f   [ Assignment ]    > CFAV{@Initialized, Object}
+(this).f   [ FieldAccess ]    > CFAV{, T}
```

Comment:
@flo2702 These changes come a bit as a surprise, but look good to me - field `f` has type `T`, so the new types look more precise than before, where it used the erasure.

Reply:
Seems like scanning through all members once in the visitor somehow allows more precise types to be used during the dataflow refinement. Probably something to consider while I look for a better fix, but I admit that I don't really understand how the visitor affects the refinement in this way.